src/GVF/GVF.jl

```diff
@@ -20,6 +20,12 @@ abstract type AbstractGVF end
 
 function get(gvf::AbstractGVF, state_t, action_t, state_tp1, action_tp1, preds_tilde) end
 
+get(gvf::AbstractGVF, state_t, action_t, state_tp1, preds_tilde) =
+    get(gvf::AbstractGVF, state_t, action_t, state_tp1, nothing, preds_tilde)
+
+get(gvf::AbstractGVF, state_t, action_t, state_tp1) =
+    get(gvf::AbstractGVF, state_t, action_t, state_tp1, nothing, nothing)
+
```

Comment:
I don't think we should be doing this for the base abstract gvf class, only for specific ones.

Reply:
That's fair, I can push a revision shortly